.github/workflows/ci_workflows.yml

```diff
@@ -143,39 +146,41 @@ jobs:
         sudo apt-get update
         sudo apt-get install language-pack-de tzdata
     - name: Install Python dependencies
-      run: python -m pip install --upgrade tox codecov
+      run: python -m pip install --upgrade tox
     - name: Run tests
       run: tox ${{ matrix.toxargs }} -e ${{ matrix.toxenv }} -- ${{ matrix.toxposargs }}
 
-  parallel_and_32bit:
-    name: 32-bit and parallel
+  parallel-double:
+    name: Parallel and double
```

Comment:
`double` refers to the repeated run? Maybe something with "repeat" or "run twice" would make this clearer; first thing I was thinking of was something with double precision (but perhaps only in memory of the 32-bit job ;-)
Having the Python version stated there would also give a better overview.
And while at that I was wondering if this job should stay at the now oldest supported Python version.

Reply:
`double` naming is irrelevant now if we end up keeping 32-bit, but I'll keep this comment in mind if we change our mind again. Thanks!

I think it is ok to stick with oldest Python version here. We already have to pin to older numpy for reasons.

Reply:
I thought the numpy version was also only pinned for the 32-bit env, but if you've managed to save that now, all the better.

Reply:
Right, I unpinned numpy too when I removed 32-bit but now it is back, so this is moot lol.